libchisel/src/remapstart.rs

```diff
@@ -0,0 +1,181 @@
+use std::collections::HashMap;
+
+use super::{ModuleError, ModulePreset, ModuleTranslator};
+use parity_wasm::elements::*;
+
+pub struct RemapStart {}
+
+impl ModulePreset for RemapStart {
+    fn with_preset(preset: &str) -> Result<Self, ()> {
+        match preset {
+            "ewasm" => Ok(RemapStart {}),
```

Comment:
This is a bit strange but also my fault because the CLI config stuff is still pretty bad.
Maybe leave a note here to change later.